lib/App.js

```diff
@@ -53,6 +55,31 @@ class App {
     this._git = new GitCommands(appPath);
   }
 
+  /* 
+  * Returns true if typescript is set to true in the app.json 
+  */
+  static usesTypeScript({ appPath }) {
+    return fs.existsSync(path.join(appPath, 'tsconfig.json'));
+  }
+
+  async convertFromTypeScript(appPath) {
+    Log(colors.green('✓ Typescript detected. Compiling...'));
+  
+    try {
+      const newPath = path.join(appPath, 'build/');
+      await fse.remove(newPath).catch(err => {});
+      await exec(`npm run ts-compiler`, { cwd: appPath });
+
+      appPath = newPath;
+      Log(colors.green('✓ Typescript compilation succesful'));
```

Comment:
```suggestion
      Log(colors.green('✓ Typescript compilation successful'));
```

maybe this log is a bit too much